ipyleaflet/leaflet.py

```diff
@@ -2072,6 +2072,8 @@ class SearchControl(Control):
         The marker used by the control.
     found_style: default {‘fillColor’: ‘#3f0’, ‘color’: ‘#0f0’}
         Style for searched feature when searching in LayerGroup.
+    property_name: default None
```

Comment:
We usually list the type and the default value. Here the type would be `string`, and it would default to `"display_name"`, not `None`.